docs/architecture.md

```
@@ -555,12 +557,16 @@ docker run -v $(pwd)/data:/data climate_indices:2.2.0 \
 
 **Trade-off**: Separate parallelization logic in CLI and xarray layers.
 
+See [ADR-0002](adr/0002-multiprocessing-cli-dask-xarray.md).
+
 ### 3. Time Dimension Chunking Constraint
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)